python/note_input_widget/ui/note_input_widget.py

```diff
@@ -30,69 +31,70 @@ def setupUi(self, NoteInputWidget):
         self.verticalLayout_2.setContentsMargins(0, 0, 0, 0)
         self.verticalLayout_2.setObjectName("verticalLayout_2")
         self.horizontalLayout = QtGui.QHBoxLayout()
+        self.horizontalLayout.setSpacing(7)
         self.horizontalLayout.setObjectName("horizontalLayout")
+        self.verticalLayout = QtGui.QVBoxLayout()
+        self.verticalLayout.setObjectName("verticalLayout")
         self.text_entry = NoteEditor(self.note_editor_page)
         self.text_entry.setFocusPolicy(QtCore.Qt.ClickFocus)
         self.text_entry.setObjectName("text_entry")
-        self.horizontalLayout.addWidget(self.text_entry)
-        self.thumbnail = QtGui.QLabel(self.note_editor_page)
-        self.thumbnail.setEnabled(True)
-        self.thumbnail.setMinimumSize(QtCore.QSize(96, 75))
-        self.thumbnail.setMaximumSize(QtCore.QSize(96, 75))
-        self.thumbnail.setText("")
-        self.thumbnail.setPixmap(QtGui.QPixmap(":/tk_framework_qtwidgets/rect_512x400.png"))
-        self.thumbnail.setScaledContents(True)
-        self.thumbnail.setAlignment(QtCore.Qt.AlignBottom|QtCore.Qt.AlignHCenter)
-        self.thumbnail.setObjectName("thumbnail")
-        self.horizontalLayout.addWidget(self.thumbnail)
-        self.button_layout_right = QtGui.QVBoxLayout()
-        self.button_layout_right.setSpacing(3)
-        self.button_layout_right.setContentsMargins(5, -1, -1, -1)
-        self.button_layout_right.setObjectName("button_layout_right")
-        self.close = QtGui.QToolButton(self.note_editor_page)
+        self.verticalLayout.addWidget(self.text_entry)
+        self.horizontalLayout_5 = QtGui.QHBoxLayout()
+        self.horizontalLayout_5.setSpacing(7)
+        self.horizontalLayout_5.setContentsMargins(-1, 3, -1, 3)
+        self.horizontalLayout_5.setObjectName("horizontalLayout_5")
+        self.screenshot = QtGui.QToolButton(self.note_editor_page)
         icon = QtGui.QIcon()
-        icon.addPixmap(QtGui.QPixmap(":/tk_framework_qtwidgets.note_input_widget/close.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
```

Comment:
Do we still use close.png anywhere? If not, we could cleanup this resource from the repo.

Reply:
We do not. I will clean this up, along with the "check mark" icon that is no longer used either.